core/client/fs/src/test/java/alluxio/client/file/cache/store/LocalPageStoreTest.java

```diff
@@ -70,19 +71,37 @@ public void testMultiFileBucket() throws Exception {
     long numFiles = numBuckets * 10;
     for (int i = 0; i < numFiles; i++) {
       PageId id = new PageId(Integer.toString(i), 0);
-      pageStore.put(id, "test".getBytes());
+      pageStore.put(id, "test".getBytes(), false);
     }
     assertEquals(10, Files.list(
             Paths.get(mOptions.getRootDir().toString(), Long.toString(mOptions.getPageSize())))
         .count());
   }
 
+  @Test
+  public void testAllTempPageWriteToTempFolder() throws Exception {
+    int numBuckets = 10;
+    mOptions.setFileBuckets(numBuckets);
+    LocalPageStore pageStore = new LocalPageStore(mOptions);
+    long numFiles = numBuckets * 10;
+    for (int i = 0; i < numFiles; i++) {
+      PageId id = new PageId(Integer.toString(i), 0);
+      pageStore.put(id, "test".getBytes(), true);
+    }
+    assertEquals(1, Files.list(
+            Paths.get(mOptions.getRootDir().toString(), Long.toString(mOptions.getPageSize())))
```

Comment:
Codify the design that the page is stored inside a directory whose name is the containing file's ID, which is again stored in a bucket dir:

```java
Path pageFile = Paths.get(pageStore.getFilePath(id, true));
Path parentFileDir = pageFile.parentDir();
Path bucketDir = parentFileDir.parentDir();
assertTrue(bucketDir.exists());
assertTrue(TEMP_DIR, bucketDir);
```

Reply:
done